test/extended/util/framework.go

```diff
@@ -320,6 +391,12 @@ func WaitForSamplesImagestream(ctx context.Context, oc *CLI, imagestream string,
 	return pollErr
 }
 
+// CustomImageStream uses the provided imageStreamObj reference to create an imagestream with the given name in the given namespace.
+func CustomImageStreams(oc *CLI, imageStream *imagev1.ImageStream) error {
```

Comment:
nit: singular instead of plural

```suggestion
func CustomImageStream(oc *CLI, imageStream *imagev1.ImageStream) error {
```